### DISCUSSION TITLE
Nextcloud app

### DISCUSSION THREAD

**Discussion by lil5** [schoentoon/go-cloud#10] [Q&A]
Is it possible to connect with the Nextcloud phone app? Or is this in the pipeline?

**Comment by schoentoon** [comment]
Way too early stage to even consider that, so the short answer is no. I had not considered compatibility with NextCloud app at all honestly, will look into whether that's possible when the time is right. If not, I am longterm planning on writing a mobile app myself for it (for Android at least, no iOS here).

**Reply by lil5** [reply]
In theory having the same endpoints should allow for a connection with the Nextcloud mobile,sidelining the need to develop one yourself, although I'd understand hesitation concerning upkeep.

I you're interested I'd like to have  a look at this as iOS + Android support are a requirement in my household.

**Reply by schoentoon** [reply]
I will keep it in mind. I however would not bother with this just yet, as it's still too early for this imo. I'll give a shout when I feel like the time is right though :)
